sdk/storage/Azure.Storage.Blobs/tests/AlwaysFailsKeyEncryptionKeyResolver.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Cryptography;
+
+namespace Azure.Storage.Blobs.Tests
+{
+    internal class AlwaysFailsKeyEncryptionKeyResolver : IKeyEncryptionKeyResolver
```

Comment:
nit: it doesn't always fail.

Btw. I'd use Moq for such simple case rather than creating new type. I believe we have it as dependency.

Reply:
addressed in upcoming push